docs/ref/contrib/admin/index.txt

```diff
@@ -2816,23 +2816,23 @@ your URLconf. Specifically, add these four patterns::
 
     from django.contrib.auth import views as auth_views
 
-    url(
-        r'^admin/password_reset/$',
+    path(
+        'admin/password_reset/',
         auth_views.PasswordResetView.as_view(),
         name='admin_password_reset',
     ),
-    url(
-        r'^admin/password_reset/done/$',
+    path(
+        'admin/password_reset/done/',
         auth_views.PasswordResetDoneView.as_view(),
         name='password_reset_done',
     ),
-    url(
-        r'^reset/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>.+)/$',
+    path(
+        'reset/<uuid:uidb64>/<token>/',
```

Comment:
Similar to the comment about changing to use `int` in generic views, this change doesn't work. The `uuid` converter doesn't match the tokens that might appear in password reset (and even if it did, the `PasswordResetConfirmView` isn't prepared to receive the parameter as a `UUID` rather than a string. The auth tests pass with `<uidb64>`.

Reply:
That's because uidb64 is a b64 encode user id, not a uuid.